base_vat_optional_vies/models/res_partner.py

```diff
@@ -99,7 +95,10 @@ def vies_vat_optional_check(self, country_code, vat_number):
             # Validate against VAT Information Exchange System (VIES)
             # see also http://ec.europa.eu/taxation_customs/vies/
             vat = country_code + vat_number
-            res = vatnumber.check_vies(vat)
+            if not self.env.context.get('test_vies'):
```

Comment:
You are ignoring config['test_enable'] as specified and still not calling super...